Core/scripts/dirac-install.py

```diff
@@ -1330,9 +1330,17 @@ def createBashrc():
         lines.append( '[ -z "$HOME" ] && export HOME=%s' % os.environ['HOME'] )
       if 'X509_CERT_DIR' in os.environ:
         lines.append( 'export X509_CERT_DIR=%s' % os.environ['X509_CERT_DIR'] )
-      elif not os.path.isdir( "/etc/grid-security/certificates" ):
-        lines.append( "[[ -d '%s/etc/grid-security/certificates' ]] && export X509_CERT_DIR='%s/etc/grid-security/certificates'" % ( proPath, proPath ) )
+        certDir = os.environ['X509_CERT_DIR']
+      else:
+        if os.path.isdir( "/etc/grid-security/certificates" ):
+          certDir = os.environ['X509_CERT_DIR']
+        else:
+          certDir = '%s/etc/grid-security/certificates' % proPath
+          lines.append( "[[ -d '%s' ]] && export X509_CERT_DIR='%s'" % ( certDir, certDir ) )
```

Comment:
I agree with you, but I didn't change it that way because I wanted to preserve the previous mode. 
Anyway, since we agree, I do as you say.